modules/ROOT/pages/lb-mapping-rules.adoc

```diff
@@ -8,165 +8,278 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
+
+[%header,cols="10a,20a,20a,20a,10a"]
+|===
+| Rule # | *Input Path* | *Target App* | *Output Path* | *Protocol*
+| 1 | / | myApp | / | http 
+|===
+
+This rule matches the `dev-misc-mythellotest-v1-myapp` app:
 
-[%header,cols="10a,20a,20a,10a"]
+[%header,cols="10a,20a,20a,20a,10a"]
 |===
-|Rule # |Input URL 2+^| Output URL
-|   |    *URI*   |       *appName*   |   *appURI*
-| 0 | /{app}/    | {app}             | /
+| Rule # | *Input Path* | *Target App* | *Output Path* | *Protocol*
+| 1 | /{layera}/{domaina}/v{versiona}/ | dev-{layera}-{domaina}-{versiona}-myapp | /api/ | http
 |===
 
-This rule maps `lb-demo.lb.anypointdns.net/{app}` to `{app}.cloudhub.io`. `{app}` is a pattern for any application name you pass in the inbound request.
 
+== Using Patterns in Mapping Rules
 
-=== Host Mapping
+A _pattern_ is a string that defines a template for matching an input text. The value in curly brackets (`{   }`) is treated as a variable.
 
-If you have a wildcard certificate (such as `*.example.com`), you can use the `subdomain` variable to map any subdomain:
+* Variable *names* can contain only lowercase letters (`a-z`) and no other characters.
+* Variable *values* can contain the characters `a-z0-1.&?-_`, but can't contain slash characters (`/`).
 
-[%header,cols="10a,20a,20a,10a"]
+
+[IMPORTANT]
+DLBs don't support patterns in the *Output Path* (`appURI`).
+
+If you set up a DNS CNAME record to map `example.com` to `_lb-name_.lb.anypointdns.net`,
+you can then use a mapping rule to map `app.example.com` to a different deployed CloudHub Mule application name.
+
+[%header,cols="10a,20a,20a,20a,10a"]
 |===
-|Rule # |Input URL 2+^| Output URL
-|   |  *URI*   |       *appName*    |  *appURI*
-| 0 | /        | {subdomain}   | /
+| Rule # | *Input Path* | *Target App* | *Output Path* | *Protocol*
+|1 | /api/v1/ | default-app | /api/v1/ | http
+|2 | /api/v1/secondapp | second-app | /api/v1	http
 |===
-This rule automatically maps any request passed to a subdomain of `example.com` to the corresponding `appName`:
 
-* Passing `api.example.com` would redirect to `api.cloudhub.io` +
-* Passing `application.example.com` is mapped to `application.cloudhub.io`.
 
-The same applies to the https://en.wikipedia.org/wiki/Subject_Alternative_Name[Subject Alternative Names] (SANs) of your SSL endpoints.
+You can route external requests to `+https://app.example.com+` to `+http://app.cloudhub.io/example+`:
 
-Instead of using a wildcard certificate, which permits any subdomain application name to be passed through the dedicated load balancer, you can restrict the allowed subdomain names. To do this, you must have different SANs configured for a certificate's common name. You can use the `subdomain` variable to map the subdomain portion of your domain name to your application.
+[%header,cols="10a,20a,20a,20a,10a"]
+|===
+| Rule # | *Input Path* | *Target App* | *Output Path* | *Protocol*
+|1 | app.example.com | app | /example | http
```

Comment:
Makes sense... haven't really tested it but it was documented as if that would work and looking at the regex it should. I'll have to validate it later on (but it looks ok based off on what was written before)

Reply:
This example came from the current docs: https://docs.mulesoft.com/runtime-manager/lb-mapping-rules#using-patterns-in-mapping-rules:

> For example, you can bind two hostnames for redirect:
> ‘app.example.com’ ->  application: `app` URI: `/example’
> In this case, external requests to https://app.example.com are directed to http://app.cloudhub.io/example.

Reply:
I know... we can leave it as is and change it when I have it tested.